packages/form-generator/src/codegen_graphql_form_generation_result.ts

```diff
@@ -33,11 +31,15 @@ export class CodegenGraphqlFormGeneratorResult
         const fd = await fs.open(filePath, 'w+');
         try {
           await fd.writeFile(content);
-          log?.(`File written: ${path.relative(process.cwd(), filePath)}`);
+          fileWrittenMessages.push(
```

Comment:
Rather than returning a string, this should return an object with information that the caller can use to print log messages. eg an object like
```ts
{
  filesWritten: string[]
}
```

Reply:
Since this type is common in multiple packages, I added a `FilesWrittenResult` type in `plugin-types` to keep them all consistent